firestore-debug.log

```diff
@@ -0,0 +1,11 @@
+Jul 10, 2021 3:27:56 PM com.google.cloud.datastore.emulator.firestore.websocket.WebSocketServer start
```

Comment:
This file should be removed and/or ignored?

Reply:
added to gitignore